### ISSUE TITLE
Can't seem to run MangoHud on a 32bit game

### ISSUE THREAD

**Issue by zastrixarundell** [flathub/org.freedesktop.Platform.VulkanLayer.MangoHud#50]
Essentially I've ran into a problem when running a 32bit games: mangohud just doesn't show. 

I've initially seen this issue with **Prince of Persia: Warrior Within** but I've tested out **Wolfenstein: Enemy Territory** because it's free and takes around 1GB so anyone can download it/test it out.

Linked issue: [#1285 on MangoHud](https://github.com/flightlessmango/MangoHud/issues/1285).

**Comment by zastrixarundell** [comment]
Hm it might not be inherently with 32bit application with itself. I've installed Fallout: New Vegas and mangohud is working properly: 
![image](https://github.com/flathub/org.freedesktop.Platform.VulkanLayer.MangoHud/assets/30195425/fec8a474-e9b5-4509-b343-00ccddc998b3)

**Comment by cold-distance** [comment]
I deleted my message because I saw that MangoHud wasn't update in my PC. Sorry.

**Comment by jntesteves** [comment]
Hi, thanks for the report. First, I want to clarify that 32-bit games work with this package. Both 64-bit and 32-bit libraries are included together in the same extension.

I currently don't have Prince of Persia: Warrior Within to test.

I've just tested Wolfenstein: Enemy Territory and I tried every LD_* library loading trick I know, and I can't make MangoHud show up in this game. I don't think this is a problem with the Flatpak package. I'll follow the upstream issue, as I think it's probably MangoHud that currently doesn't work with this game. If there's confirmation it works outside of Flatpak and only fails with this package I'll investigate further.

I'll close this issue and follow flightlessmango/MangoHud#1285 for now. We can reopen if there's action to be taken here.
